server/neptune/workflows/internal/deploy/revision/queue/deployer.go

```diff
@@ -89,6 +89,8 @@ func (p *Deployer) Deploy(ctx workflow.Context, requestedDeployment terraformWor
 		logger.Error(ctx, "unable to persist deployment, proceeding with in-memory value", key.ErrKey, persistErr)
 	}
 
+	// TODO: Add pullRebaser.RebaseOpenPRs() with versioning
```

Comment:
Since you are moving forward with splitting out the rebase logic, how about define the `PullRebaser` in the `Worker` and run the routine once the deploy step is complete? Since it shouldn't determine whether or not a deploy is successful anyways

Reply:
yeah I think that makes sense, removed the comment!